tests/test_server_agent.py

```diff
@@ -2097,6 +2179,11 @@ def assert_handler_failed(resource, msg):
 
 
 @pytest.mark.asyncio(timeout=15)
+@pytest.mark.skip(reason="""Rewrite when a full deploy can be requested from the server.
```

Comment:
I think xfail is the more appropriate annotation

https://docs.pytest.org/en/latest/skipping.html#xfail-mark-test-functions-as-expected-to-fail